docs/make.jl

```diff
@@ -8,6 +8,7 @@ using Documenter
 examples = [
     "Lorenz example" => "examples/lorenz_example.md",
     "Turbulence example" => "examples/edmf_example.md",
+    "Sinusoid example" => "examples/sinusoid_example.md",
```

Comment:
Can we call this "Simple example walkthrough" and move it to the top of the `examples[...]` list (so it is the first examples page)